packages/cli/config-management/docs-configuration/src/DocsConfiguration.ts

```diff
@@ -9,13 +9,37 @@ export interface DocsConfiguration {
     favicon: ImageReference | undefined;
     colors: FernRegistry.docs.v1.write.ColorsConfig;
     navbarLinks: FernRegistry.docs.v1.write.NavbarLink[] | undefined;
+    typography: TypographyConfig | undefined;
 }
 
 export interface Logo {
     reference: ImageReference;
     href: FernRegistry.docs.v1.write.Url | undefined;
 }
 
+export interface FontConfig {
+    path: string;
+    name?: string;
```

Comment:
Nit but for these configs I always do `name: string | undefined` rather than `name?: string` to ensure we don't forget it in the conversion. same for inside the `TypographyConfig`

Reply:
Gotcha! Done!